duffy/api_v2/credentials.py

```diff
@@ -0,0 +1,11 @@
+import os
+def get_nova_credentials():
```

Comment:
I like this, and won't block merge, but can we do 2 things:
1.) let's move this out of the view and into util or perhaps the model 
and
2.) can we grow a feature to set this information in a configuration file (like `/etc/duffy.conf`)?

Reply:
using 2 (through /etc/duffy,conf)